source/globalVars.py

```diff
@@ -48,41 +53,51 @@ class DefaultAppArgs(argparse.Namespace):
 	easeOfAccess: bool = False
 
 
-startTime=0
+# Encapsulated by api module,
+# refer to #14037 for removal strategy.
 desktopObject: typing.Optional['NVDAObjects.NVDAObject'] = None
+"""Deprecated, use `setDesktopObject|getDesktopObject` from `api` instead"""
 foregroundObject: typing.Optional['NVDAObjects.NVDAObject'] = None
+"""Deprecated, use `setForegroundObject|getForegroundObject` from `api` instead"""
 focusObject: typing.Optional['NVDAObjects.NVDAObject'] = None
+"""Deprecated, use `setFocusObject|getFocusObject` from `api` instead"""
 focusAncestors: typing.List['NVDAObjects.NVDAObject'] = []
+"""Deprecated, use `getFocusAncestors` from `api` instead"""
 focusDifferenceLevel=None
+"""Deprecated, use `getFocusDifferenceLevel` from `api` instead"""
 mouseObject: typing.Optional['NVDAObjects.NVDAObject'] = None
-mouseOldX=None
-mouseOldY=None
+"""Deprecated, use ``setMouseObject|getMouseObject`` from `api` instead"""
 navigatorObject: typing.Optional['NVDAObjects.NVDAObject'] = None
+"""Deprecated, use ``setNavigatorObject|getNavigatorObject`` from `api` instead"""
 reviewPosition=None
+"""Deprecated, use ``getReviewPosition|setReviewPosition`` from `api` instead"""
 reviewPositionObj=None
-lastProgressValue=0
+"""Deprecated, use ``api.getReviewPosition().obj`` instead"""
+
+# unused, should eventually get removed.
+mouseOldX = None
+"""Deprecated, this is unused and not set by NVDA core"""
+mouseOldY = None

```

Comment:
Please add the typing (`int`) here (it was in the doc-string previously). Any others you can easily get the type for would be helpful too.

Reply:
Considering they are unused and never set to `int` I think this is misleading.

Perhaps this instead?

```suggestion
mouseOldX: Literal[None] = None
"""Deprecated, this is unused and not set by NVDA core"""
mouseOldY: Literal[None]  = None
```

Reply:
Yep, that seems fine.